utils/severityutils/severity.go

```diff
@@ -30,6 +33,45 @@ const (
 	Unknown  Severity = "Unknown"
 )
 
+func GetSeverityIcon(severity Severity, svg bool) string {
+	if svg {
+		return getSeveritySvgIcon(severity)
+	}
+	return getSeverityEmojiIcon(severity)
+}
+
+func getSeverityEmojiIcon(severity Severity) string {
+	switch severity {
+	case Critical:
+		return "❗️"
+	case High:
+		return "🔴"
+	case Medium:
+		return "🟠"
+	case Low:
+		return "🟡"
+	default:
+		return "⚪️"
+	}
+}
+
+func getSeveritySvgIcon(severity Severity) string {
```

Comment:
I don't see we ever use this - traced back and the only call to it gets "false"...

Reply:
removed